exercises/simple-linked-list/example.js

```diff
@@ -0,0 +1,72 @@
+function ListElement (value, next) {
+  if (!(this instanceof ListElement))
+    throw new Error('ListElement is a constructor.');
+
+  if (value === undefined)
+    throw new Error('Value required.');
+
+  if (next !== undefined && !(next instanceof ListElement))
+    throw new Error('A ListElement instance as next value is required.');
+
+  this.value = value;
+  this.next = next;
+}
+
+function LinkedList () {
+}
+
+LinkedList.prototype.push = function (value) {
+  if (value === undefined) throw new Error('Argument required.');
+
+  var newEl = (value instanceof ListElement)
+    ? value
+    : new ListElement(value)
+
+  if (!this.head) {
+    this.head = newEl;
+    return;
+  }
+
+  var lastEl = this.head;
+  while (lastEl.next) lastEl = lastEl.next;
+
+  lastEl.next = newEl;
+}
```

Comment:
Semicolon after here too.  Again, I won't call it out everywhere, but our current style is to use them by default.